scipy/stats/_stats.pyx

```diff
@@ -584,3 +590,177 @@ def gaussian_kernel_estimate(points, values, xi, precision, dtype, real _=0):
                 estimate[j, k] += values_[i, k] * arg
 
     return np.asarray(estimate)
+
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+def _rvs_argus_rejection_beta(double chi, Py_ssize_t n, rand_state):
```

Comment:
Have you looked through the annotations of this cython code to see if there are Python dependent calls that can be removed?